hackathon_site/event/jinja2/event/landing.html

```diff
@@ -12,299 +14,365 @@
+                                </span>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                What prizes will be available?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    This year, 1st, 2nd, and 3rd place teams overall will receive cash prizes of $2500, $1500, and $750 respectively, while the winning teams for our design challenges will receive $750.
+                                </span>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                When can I expect to hear back?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    Acceptances would be sent out on a rolling basis, so apply early! 
+                                </span>
+                            </div>
+                        </li>
+                    </ul>
                 </div>
-                <div class="scheduleRow">
-                    <p class="scheduleTime">4PM</p>
-                    <p>End</p>
+
+                <div class="col s12 m6">
+                    <ul class="collapsible">
+                        <li>
+                            <div class="collapsible-header">
+                                How does MakeUofT work?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    This year MakeUofT has changed its format to accommodate the difficulties of hosting a virtual makeathon. For MakeUofT 2021, instead of a traditional makeathon, you get to spend up to 2 weeks on the project you are working on and submit it! The challenges for MakeUofT would be announced during the event’s opening ceremony on February 6th, 2021. 
+                                </span>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                Who can apply? When is the deadline to apply?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>
+                                    We’re welcoming any students of age 18 or over pursuing post-secondary education! Applications open on January 7th, 2021 and close at 11:59 PM EST on February 4th. Acceptances will be internally decided on a rolling basis, so apply early to maximize your chances of getting accepted!
```

Comment:
According to the code, they open January 6 and close February 5 at 11:59. If logistics wants it to be the 7th-4th, we should update the settings file

Reply:
I'll ask them

Reply:
"Jan 7th to Feb 4th would be better for us in order to for us to deal with the last minute applications"

Reply:
Sounds good, can you update the settings file to match?

Reply:
Scratch that, you already did!